ner_v1/detectors/textual/text/text_detection.py

```diff
@@ -154,20 +154,24 @@ def set_min_token_size_for_levenshtein(self, min_size):
         """
         self._min_token_size_for_fuzziness = min_size
 
-    def _process_text(self, text):
-        self.text = text.lower()
-        if isinstance(self.text, bytes):
-            self.text = self.text.decode('utf-8')
+    def _process_text(self, texts):
+        text_lowercase = [text.lower() for text in texts]
+        self.text = []
+        for text in text_lowercase:
+            if isinstance(text, bytes):
+                self.text.append(text.decode('utf-8'))
+            else:
+                self.text.append(text)
 
         self.processed_text = self.text
 
         # Note: following rules have been disabled because cause problem with generating original text
-        # regx_to_process = RegexReplace([(r'[\'\/]', r''), (r'\s+', r' ')])
+        # regex_to_process = RegexReplace([(r'[\'\/]', r''), (r'\s+', r' ')])
         # self.processed_text = self.regx_to_process.text_substitute(self.processed_text)
-        self.processed_text = u' ' + self.processed_text + u' '
+        self.processed_text = [u' ' + processed_text + u' ' for processed_text in self.processed_text]
         self.tagged_text = self.processed_text
 
-    def _get_substring_from_processed_text(self, matched_tokens):
+    def _get_substring_from_processed_text(self, matched_tokens, sentence):
```

Comment:
Change this to  `_get_substring_from_processed_text(self, text, matched_tokens)`
Change the internal functions (`_get_tokens_and_indices`) `text` argument to `txt`

Reply:
Done